schedule/reschedule.py

```diff
@@ -72,12 +74,14 @@ def apply_fuzz(self, ivl):
                 return int(self.fuzz_factor * (max_ivl - min_ivl + 1) + min_ivl)
         else:
             min_num_cards = 18446744073709551616
-            best_ivl = (max_ivl + min_ivl) // 2
+            best_ivl = (max_ivl + min_ivl) // 2 if self.allow_to_past else max_ivl
```

Comment:
What does best_ivl actually do? Its exact function is difficult to decipher.

Reply:
It sets a default value for best_ivl.